exonum-node/src/events/tests.rs

```diff
@@ -231,49 +230,49 @@ impl HandshakeParams {
     // default `max_message_len`.
     #[doc(hidden)]
     pub fn with_default_params() -> Self {
-        let (public_key, secret_key) = gen_keypair_from_seed(&Seed::new([1; SEED_LENGTH]));
+        let keys = KeyPair::from_seed(&Seed::new([1; SEED_LENGTH]));
         let address = "127.0.0.1:8000";
 
         let connect = Verified::from_value(
             Connect::new(address, SystemTime::now().into(), &user_agent()),
-            public_key,
-            &secret_key,
+            keys.public_key(),
+            keys.secret_key(),
         );
 
         let mut params = HandshakeParams::new(
-            public_key,
-            secret_key.clone(),
+            keys.public_key(),
+            keys.secret_key().clone(),
             SharedConnectList::default(),
             connect,
             ConsensusConfig::DEFAULT_MAX_MESSAGE_LEN,
         );
 
-        params.set_remote_key(public_key);
+        params.set_remote_key(keys.public_key());
         params
     }
 }
 
 impl ConnectionParams {
     fn from_address(address: SocketAddr) -> Self {
-        let (public_key, secret_key) = gen_keypair();
-        let connect = connect_message(address, public_key, &secret_key);
+        let keys = KeyPair::random();
+        let connect = connect_message(address, keys.public_key(), keys.secret_key());
         let handshake_params = HandshakeParams::new(
```

Comment:
Nit: It could make sense to change `HandshakeParams::new` to take `KeyPair` instead of separate keys.